src/server/server_family.cc

```diff
@@ -1882,6 +1855,56 @@ void ServerFamily::Auth(CmdArgList args, const CommandContext& cmd_cntx) {
   }
 }
 
+void ServerFamily::ClientUnPauseCmd(CmdArgList args, SinkReplyBuilder* builder) {
```

Comment:
When we run client unpause after client pause timout ALL how do you make sure that this command is invoked?  we have mechanism to wait on all connections untill pause is done

Reply:
>When we run client unpause after client pause timout ALL how do you make sure that this command is invoked?  

It's not allowed to call `CLIENT UNPAUSE` if a `CLIENT PAUSE` preceded the UNPAUSE.

Reply:
Oh I see what you are asking with 

> we have mechanism to wait on all connections untill pause is done

Yes we should block and then execute this. I don't see why we would have an issue but I will also add a test for this :)